frontend/components/StudentOverview/StudentOverview.tsx

```diff
@@ -32,35 +33,123 @@ export const StudentOverview: React.FC<{ student: Student }> = ({
         }
     };
 
-    // eslint-disable-next-line @typescript-eslint/no-unused-vars
-    const buttonHandler = (event: React.MouseEvent<HTMLButtonElement>) => {
-        console.log("hallo");
+    const yesSuggest = () => {
+        makeSuggestion("YES");
+    };
+
+    const maybeSuggest = () => {
+        makeSuggestion("MAYBE");
+    };
+
+    const noSuggest = () => {
+        makeSuggestion("NO");
+    };
+
+    const definitiveYes = () => {
+        makeDecision("YES");
+    };
+
+    const definitiveNo = () => {
+        makeDecision("NO");
+    };
+
+    const definitiveMaybe = () => {
+        makeDecision("MAYBE");
+    };
```

Comment:
There should be an enum that you can use for this.

Reply:
That would be cleaner, I will look into it.

Reply:
This should be fixed now